.github/workflows/notebooks.yml

```diff
@@ -18,4 +19,4 @@ jobs:
         with:
           python-version: '3.11'
           env-files: .ci_support/environment-notebooks-integration.yml
-          exclusion-file: .ci_support/exclude
\ No newline at end of file
+          exclusion-file: .ci_support/exclude
```

Comment:
I didn't touch this line... But I guess it only added EOF so I don't think it disturbs